src/pages/dar_application/ResearcherInfo.jsx

```diff
@@ -0,0 +1,183 @@
+import { useState, useEffect } from 'react';
+import { Alert } from '../../components/Alert';
+import { Link } from 'react-router-dom';
+import { a, div, fieldset, h, h3, input, label, span } from 'react-hyperscript-helpers';
+import { eRACommons } from '../../components/eRACommons';
+
+const profileLink = h(Link, {to:'/profile', className:'hover-color'}, ['Your Profile']);
+
+const profileUnsubmitted = span(["Please submit ", profileLink, " to be able to create a Data Access Request"]);
+
+const profileSubmitted = span(["Please make sure ", profileLink, " is updated as it will be used to pre-populate parts of the Data Access Request"]);
+
+export default function ResearcherInfo(props) {
+  //NOTE: seems like we'll be passing in a lot of prop data to this individual component due to conditional rendering
+  //a lot of the variables needed are read only values (for step 1 at least), so they should remain props
+  //remainder are either update functions from the parent or prop values needed to initialize state values
+  //raises question on whether or not it can be broken down further or if data flow needs to be re-evaluated due to shift from monolith to compnents
+  const [checkCollaborator, setCheckCollaborator] = useState(props.checkCollaborator);
+
+  useEffect(() => {
+    setCheckCollaborator(props.checkCollaborator);
+  }, [props.checkCollaborator]);
+
+  //helper function to coordinate local state changes as well as updates to form data on the parent
+  const formStateChange = (stateVarSetter, attr, event) => {
+    const name = event.target.name;
+    const value = event.target[attr];
+    //NOTE: come up with a better name for this function, sounds too similar to formStateChange
+    props.formFieldChange(name, value);
+    stateVarSetter(value);
+  };
+
+  return (
+    div({ className: 'col-lg-10 col-lg-offset-1 col-md-12 col-sm-12 col-xs-12' }, [
+      fieldset({ disabled: props.darCode !== null }, [
```

Comment:
I like https://lodash.com/docs/#isNil for these kinds of checks - it's mostly a matter of style.